hackathon_site/hardware/serializers.py

```diff
@@ -44,3 +44,11 @@ class Meta:
     @staticmethod
     def get_unique_hardware_count(obj: Category):
         return Hardware.objects.filter(categories__id=obj.id).count()
+
+
+class HardwareOrderSerializer(serializers.ModelSerializer):
```

Comment:
This can just be called `OrderSerializer` to stick with our convention of naming it after the model

Reply:
renamed. also renamed the tests